internal/domain/pull_request.go

```diff
@@ -0,0 +1,30 @@
+package domain
```

Comment:
Document the struct types and their fields with comments to improve readability and maintainability. This helps other developers (or future you) understand the purpose and constraints of each field without having to reference external documentation.